### PR TITLE
Base estructure of the frontend

### LINKED ISSUE #3: Maquetation of the employee check in/out, admin, branches, employees screens with html,css and javascript
# REQUIREMENTS
A check in / out screen is required showing a clock for check-in and check-out 

# DESIGN
imagen

### PR DESCRIPTION
**Pull request by YxMARCKxY**
- We transferred the Figma design of the employee check in/out screen to HTML and CSS
- We use Javascript to create the live clock
	- The function 'startTime()' and 'checkTime()' were created to inject the time
	  obtained from the system through the Date() class in the HTML

# Description

We added the first changes/estructure according to the frontend mockups

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+  font-weight: 700;
+  cursor: pointer;
+  box-shadow: rgba(0, 0, 0, 0.12) 0px 1px 3px, rgba(0, 0, 0, 0.24) 0px 1px 2px;
+}
+
+/* .content-nogrid {
+  display: grid;
+  grid-template-columns: 25% 71%;
+  margin-top: 7rem;
+} */
```
Comment (JuanequeX) [inline css/styles.css]: If this code are not necessary, can you remove it, what do you think?

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+  background-color: #2c3841;
+  box-shadow: rgba(0, 0, 0, 0.12) 0px 1px 3px, rgba(0, 0, 0, 0.24) 0px 1px 2px;
+  color: #90969b;
+  text-align: center;
+}
+
+select > option {
+background-color: white;
+color: #2c3841;
+}
```
Comment (JuanequeX) [inline css/styles.css]: Add a blank line here

File: index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      </div>
+    </div>
+  </div>
+
+
+<script src="js/timer-script.js"></script>
+
+</body>
+
+</html>
```
Comment (JuanequeX) [inline index.html]: Add blank line here

**Review by JuanequeX — CHANGES_REQUESTED** [review]
It is good practice to order the lines of code in your style blocks in alphabetical order, I recommend that you sort it.

in visual studio code can use `ctrl + shift + p` and select the option `sort lines ascending` for to do

**Comment by JuanequeX** [comment]
Just add the co-authors into the commit, you don't need to put it in your pull request description

File: css/styles.css (hunk 1 above)
Comment (YilKanda) [reply css/styles.css]: Totally agree with @JuanequeX

File: js/timer-script.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  let h = today.getHours();
+  let m = today.getMinutes();
+  let s = today.getSeconds();
+  h = checkTime(h);
+  m = checkTime(m);
+  s = checkTime(s);
+  document.getElementById('h').innerHTML = h + ":";
+  document.getElementById('m').innerHTML = m + ":";
+  document.getElementById('s').innerHTML = s;
+  //document.getElementById('txt').innerHTML =  h + ":" + m + ":" + s;
```
Comment (YilKanda) [inline js/timer-script.js]: If you don't occupy this line, delete it.

File: js/timer-script.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      hour.innerHTML = "PM"
+    }
+  }
+
+}
+
+function checkTime(i) {
+  if (i < 10) { i = "0" + i };  // add zero in front of numbers < 10
+  return i;
+}
```
Comment (YilKanda) [inline js/timer-script.js]: Add a blank line.

**Review by YilKanda — COMMENTED** [review]
I would recommend focusing on the backend part first and then the frontend part. 

It is not bad that you use a template, but it is important that you know what that template does and see if you can do something similar with a lesser amount of code, that is, optimize it.

Finally, I would like to add a phrase from Addy Osmani:

"First do it, then do it right, then do it better."

As a tip, first carry out the functionality of the project (backend), and then focus on making it look nice (frontend).

File: css/styles.css [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+  font-size: 3rem;
+}
+
+/*#arrow_left{
+ 
+}
+
+#arrow_right{
+  
+}*/
```
Comment (YilKanda) [inline css/styles.css]: If you don't occupy this code, delete it.

File: views/employees.html [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+    </center>
+    <div id="arrow_right" ><i class="fas fa-chevron-right"></i></div>
+    
+    </div>
+    
+  </div>
+
+
+
+
```
Comment (YilKanda) [inline views/employees.html]: Remove unnecessary spaces,

File: views/employees.html [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+</body>
+
+</html>
```
Comment (YilKanda) [inline views/employees.html]: At the end of each file, leave a blank line.

**Review by YilKanda — COMMENTED** [review]
Check all your files, in which you have unnecessary blank spaces.

Al the end of each file, leave a blank line.

You can check your * .html files with [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).

**Review by YilKanda — APPROVED** [review]

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by EduardoGHdez — APPROVED** [review]

**Pull request merged**